lib/Backends/CPU/libjit/libjit_conv.cpp

```diff
@@ -234,9 +236,18 @@ void libjit_convDKKC8_f(float *outW, const float *inW, const float *filterW,
                         const float *biasW, const size_t *outWdims,
                         const size_t *inWdims, const size_t *filterWdims,
                         const size_t *biasWdims, size_t filterSize,
-                        size_t stride, size_t pad, unsigned pixelScanFirst,
-                        unsigned numDepthRegs, unsigned sizeGroupY,
-                        unsigned depthStrips) {
+                        size_t stride, size_t pad, size_t group,
+                        unsigned pixelScanFirst, unsigned numDepthRegs,
+                        unsigned sizeGroupY, unsigned depthStrips) {
+  size_t inChannels = inWdims[3];
+  size_t outChannels = outWdims[3];
+  assert(inChannels % group == 0 &&
```

Comment:
Please don't use asserts in libjit. The code is compiled in debug mode and the asserts either remain in the code or are not effective, depending on how we set the optimization pipeline.

Reply:
Got it. Btw, there are asserts in other functions of this file. Will submit another PR to clean up the assert.

Reply:
see also my comment about validating these params in instr verify.